src/sentry/integrations/types.py

```diff
@@ -51,6 +51,8 @@ class ExternalProviderEnum(Enum):
     ExternalProviderEnum.CUSTOM: ExternalProviders.CUSTOM,
 }
 
+EXTERNAL_PROVIDERS_REVERSE_VALUES = {k.value: v for k, v in EXTERNAL_PROVIDERS_REVERSE.items()}
```

Comment:
created this temporarily, will eventually put a wrapper function/remove this once we completely get rid of old imports.